build_system/src/main.rs

```diff
@@ -26,17 +26,29 @@ macro_rules! arg_error {
 fn usage() {
     println!(
         "\
-Available commands for build_system:
+rustc_codegen_gcc build system
 
-    cargo     : Run cargo command
-    rustc     : Run rustc command
-    clean     : Run clean command
-    prepare   : Run prepare command
-    build     : Run build command
-    test      : Run test command
-    info      : Run info command
-    clone-gcc : Run clone-gcc command
-    --help    : Show this message"
+Usage: build_system [command] [options]
+
+Options:
+        -h, --help    : Displays this help message.
+
+Commands:
+
+        cargo     : Executes a cargo command. Use 'cargo --help' for a list of cargo commands.
+        clean     : Cleans the build directory, removing all compiled files and artifacts.
+        prepare   : Prepares the environment for building, including fetching dependencies and setting up configurations.
+        build     : Compiles the project. Use 'build --help' for build options.
+        test      : Runs tests for the project. Use 'test --help' for test options.
+        info      : Displays information about the build environment and project configuration.
+        clone-gcc : Clones the GCC compiler from a specified source. Use 'clone-gcc --help' for options.
+
+Examples:
```

Comment:
I don't think having examples in a good idea here. Better let people check the `Readme.md` file instead.

Reply:
oh, alright